makefiles/vars.inc.mk

```diff
@@ -77,6 +77,7 @@ export FLASH_ADDR            # Define an offset to flash code into ROM memory.
 export PORT                  # The port to connect the TERMPROG to.
 export ELFFILE               # The unstripped result of the compilation.
 export HEXFILE               # The stripped result of the compilation.
+# FLASHFILE                  # The output file used for flashing (transition phase: only if defined)
```

Comment:
Feel free to write as many lines here as you see necessary. Eventually this should end up in the docs.